docs/src/tutorials/dependency-updates.md

```diff
@@ -0,0 +1,313 @@
+        update:
+            command: |
+                set -e
+                bundle update --all
+                git add Gemfile Gemfile.lock
+                git add -A
+                git diff-index --quiet HEAD || git commit --allow-empty -m "Update Ruby dependencies"
+
+{{< /codetabs >}}
+<!--vale on -->
+
+## 2. Automate your dependency updates with a cron job
+
+After you've defined a source operation to [update your dependencies on your project](#1-define-a-source-operation-to-update-your-dependencies),
+you can automate it using a cron job.
+
+Note that it’s best not to run source operations on your production environment,
+but rather on a dedicated environment where you can test changes.
+
+Make sure you have the [Platform.sh CLI](../administration/cli/_index.md) installed
+and [an API token](../administration/cli/api-tokens.md#2-create-a-platformsh-api-token)
+so you can run a cron job in your app container.
+
+1. Set your API token as a top-level environment variable:
+
+{{< codetabs >}}
+
++++
+title=From the CLI
++++
+
+Run the following command:
+
+```bash
+platform variable:create --environment main --level environment --prefix 'env' --name PLATFORMSH_CLI_TOKEN --sensitive true --value 'YOUR_PLATFORMSH_CLI_TOKEN' --inheritable false --visible-build true --json false --enabled true --visible-runtime true
+```
+
+<--->
++++
+title=From the Console
++++
+
+1. Open the environment where you want to add the variable.
+2. Click {{< icon settings >}} **Settings**.
+3. Click **Variables**.
+4. Click **+ Add variable**.
+5. In the **Variable name** field, enter `env:PLATFORMSH_CLI_TOKEN`.
+6. In the **Value** field, enter your API token.
+7. Make sure the **Available at runtime** and **Sensitive variable** options are selected.
+8. Click **Add variable**.
+
+{{< /codetabs >}}
+
+{{< note theme="warning" >}}
+
+Once you add the API token as an environment variable,
+anyone with [SSH access](../development/ssh/_index.md) can read its value.
+Make sure you carefully check your [user access on this project](../administration/users.md#manage-project-users).
+
+{{< /note >}}
+
+2. Add a build hook to your app configuration to install the CLI as part of the build process:
+
+```yaml {location=".platform.app.yaml"}
+hooks:
+    build: |
+        set -e
+        echo "Installing Platform.sh CLI"
+        curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
+
+        echo "Testing Platform.sh CLI"
+        platform
+```
+
+3. Then, to configure a cron job to automatically update your dependencies once a day,
+   use a configuration similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+crons:
+    update:
+        # Run the code below every day at midnight.
+        spec: '0 0 * * *'
+        commands:
+            start: |
+                set -e
+                platform sync -e development code data --no-wait --yes
+                platform source-operation:run update --no-wait --yes
```

Comment:
```suggestion
                platform source-operation:run update-file --no-wait --yes
```

Reply:
To match this reference https://github.com/platformsh/platformsh-docs/blob/e4b79172555d22f95b2f74809088d81f3505a517/docs/src/tutorials/dependency-updates.md?plain=1#L207

Reply:
Agreed 👍

Reply:
🤦‍♀️ Thanks!

Reply:
I also updated the examples in the actual **[Define a source operation to update your dependencies](https://automated-dependency-2hxm7za-652soceglkw4u.eu-3.platformsh.site/tutorials/dependency-updates.html#1-define-a-source-operation-to-update-your-dependencies)** so hopefully everything is consistent now.

Reply:
Perfect!

Reply:
Actually @AnouckColson I think for this examples (update dependencies) it makes sense to keep the[ name of the command ](https://automated-dependency-2hxm7za-652soceglkw4u.eu-3.platformsh.site/tutorials/dependency-updates.html#1-define-a-source-operation-to-update-your-dependencies) and it's[ matching call](https://automated-dependency-2hxm7za-652soceglkw4u.eu-3.platformsh.site/tutorials/dependency-updates.html#2-automate-your-dependency-updates-with-a-cron-job) as `update`:

```yaml
crons:
    update:
        # Run the code below every day at midnight.
        spec: '0 0 * * *'
        commands:
            start: |
                set -e
                platform sync -e development code data --no-wait --yes
                platform source-operation:run update --no-wait --yes  
```

Updating a file is a simple example for the main Source Ops page, and it makes sense there. But for this page, we're not updating a file in the same context, but updating a bunch of dependencies potentially.

If we could switch these references back to simply `update`, I think this PR is good to merge!